docs/api/usage.rst

```diff
@@ -74,8 +74,8 @@ model, create a ``TypeVar`` object using ``Concrete.type_var``:
 Concrete.cast_as_concrete
 -------------------------
 
-To type narrow an object as a concrete descendent of that object, the ``Concrete.cast_as_concrete``
-may be used:
+To type narrow an object as a concrete descendent of that object, the
+``Concrete.cast_as_concrete`` may be used:
```

Comment:
> Linting the line length according to the rst Python documentation. Thought not generally followed by packages, might be a good pattern. Source: https://devguide.python.org/documentation/markup/ The only change I applied was:

    The maximum line length is 80 characters for normal text, but
    tables, deeply indented code samples and long links may extend beyond
    that. Code example bodies should use normal Python 4-space
    indentation.